### PR TITLE
Lab 2 Part 3

### PR DESCRIPTION
**Pull request by Henchel-Santillan**
Deliverables for Lab 2 Part 3

### CODE REVIEW

**Comment by Henchel-Santillan** [comment]
I used 1e-4 as the error threshold / tolerance, lmk if I should lower it

File: localization.py [hunk 1]
```diff
@@ -16,33 +16,38 @@ def __init__(self, localizationType=rawSensor):
 
         super().__init__("localizer")
         
-        # TODO Part 3: Define the QoS profile variable based on whether you are using the simulation (Turtlebot 3 Burger) or the real robot (Turtlebot 4)
+        # Part 3: Define the QoS profile variable based on whether you are using the simulation (Turtlebot 3 Burger) or the real robot (Turtlebot 4)
         # Remember to define your QoS profile based on the information available in "ros2 topic info /odom --verbose" as explained in Tutorial 3
-
-        odom_qos=...
+        odom_qos=QoSProfile(reliability = 2, durability=2, history=1, depth=10)
```
Comment (Henchel-Santillan) [inline localization.py]: Will fix

Comment (Henchel-Santillan) [reply localization.py]: Fixed in [36c42f2](https://github.com/DiegoJohnson25/MTE544_group7/pull/1/commits/36c42f25f7c92cfbdc38570cae5b0d525fbab86e)

File: decisions.py [hunk 2]
```diff
@@ -61,20 +61,30 @@ def __init__(self, publisher_msg, publishing_topic, qos_publisher, goalPoint, ra
...
         
-        # TODO Part 3: Check if you reached the goal
+        # Part 3: Check if you reached the goal
+        error_threshold = 1e-4
+
         if type(self.goal) == list:
-            reached_goal=...
-        else: 
-            reached_goal=...
+            # Trajectory: get the last point in the list of points, and check error threshold
```
Comment (Henchel-Santillan) [inline decisions.py:78]: Might need to check error for every point? I think that would make more sense?

**Review by DiegoJohnson25 — APPROVED** [review]
Looks good

**Pull request merged**